src/renderers/dom/shared/validateDOMNesting.js

```diff
@@ -316,6 +318,49 @@ if (__DEV__) {
     return stack;
   };
 
+  var getOwnerInfo = function(childInstance, childTag, ancestorInstance, ancestorTag, isParent) {
```

Comment:
This is the Stack-specific part extracted from main function.
In Fiber instances are null and we use the current element stack instead.